GitUI/Avatars/InitialsAvatarProvider.cs

```diff
@@ -12,14 +12,17 @@ namespace GitUI.Avatars
     public class InitialsAvatarProvider : IAvatarProvider
     {
         private int _unkownCounter = 0;
-        private static readonly char[] _emailInitialSeparator = new[] { '.', '-', '_' };
+        private static readonly char[] _emailInitialSeparator = ['.', '-', '_'];
+        private FontFamily _fontFamily = AppSettings.CommitFont.FontFamily;
```

Comment:
Are you sure that this is the right font?
The commit _message_ is displayed using the `CommitFont`.
`FixedWidthFont` is used for viewing diffs / files, which is the "Code font" in the settings page.

Reply:
It doesn't really matter but in fact maybe it make more sense to use "Application font". Replaced.